library/src/options.cxx

```diff
@@ -136,6 +136,21 @@ std::pair<std::string, unsigned int> options::getClosestOption(const std::string
   return ret;
 }
 
+bool options::isOptional(const std::string& option) const
+{
+  return options_tools::isOptional(option);
+}
+
+void options::reset(options& opt, const std::string& name)
+{
+  options_tools::reset(*this, name);
+}
+
+void options::removeValue(options& opt, const std::string& name)
+{
+  options_tools::removeValue(*this, name);
```

Comment:
you can implement removeValue using isOptional and reset instead of adding another generated code

Reply:
done.